tests/test_async_client.py

```diff
@@ -0,0 +1,142 @@
+import unittest
+from unittest.mock import AsyncMock, patch
+import unittest.async_case
+import willofsteel
+from willofsteel.async_client import AsyncClient
+from willofsteel.exceptions import *
+from willofsteel.types import *
+
+
+class TestAsyncClient(unittest.async_case.IsolatedAsyncioTestCase):
+
+    def setUp(self):
+        patcher = patch(
+            'willofsteel.async_client.AsyncClient.request', new_callable=AsyncMock)
+        self.addCleanup(patcher.stop)
+        self.mock_request = patcher.start()
+        self.mock_request.return_value = {}, 200
+        self.client = AsyncClient('valid_key')
+
+    @patch('willofsteel.async_client.AsyncClient.request', new_callable=AsyncMock)
+    async def test_validate_key_valid_key_raises_no_exception(self, mock_get):
+        mock_get.return_value = {}, 200
```

Comment:
```suggestion
        mock_get.return_value = {"success": True, "user_id": 0}, 200
```

Incorrect return JSON, the API returns:

```json
{
    "success": true,
    "user_id": 1234567890
}
```

Please fix this otherwise the test should fail.